### PR TITLE
Add Ex command interface

### PR DESCRIPTION
**Pull request by tomtomjhj**

### CODE REVIEW

File: autoload/sentencer.vim [hunk 1]
```diff
@@ -159,18 +159,26 @@ function! s:split(line, indent1, indent, o) abort
...
+      let l:end = l:start + v:count - 1
+  endif
   let l:orig = getline(l:start, l:end)
 
   let l:lines = []
   for [l:indent1, l:indent, l:blank, l:para] in s:paragraphs(l:orig, l:o)
     if l:blank
       let l:lines += ['']
     endif
+    if len(l:para) == 0
```
Comment (whonore) [inline autoload/sentencer.vim]: If you have a specific example in mind that requires this check would you mind adding it as a test?

**Comment by whonore** [comment]
Whoops, didn't mean to push the README change to this branch. Oh well, it's very minor so I'll just leave it.

File: autoload/sentencer.vim (hunk 1 above)
Comment (tomtomjhj) [reply autoload/sentencer.vim]: The first element of the list returned by `s:paragraphs` is always `[-1, -1, 0, []]`, which causes "list index out of range" error in `l:para[0]`. But somehow, the error is silently ignored when the function is run from formatexpr.

Comment (whonore) [reply autoload/sentencer.vim]: Ah, I see. That's annoying that it suppresses error messages, `indentexpr` does too I think. Then my only suggestion is to use `empty()` instead to check for that case.

**Pull request merged**